### PR TITLE
added check that niwot.out hasn't changed to github actions

### PR DESCRIPTION
**Pull request by dlebauer**
Following up on #17, adds check that niwot.out hasn't changed using `git diff --exit-code`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,3 +44,12 @@ jobs:
       run: |
         echo "::error title={No Output}::Test run for Niwot site failed to produce output"
         exit 1
+    # check whether niwot.out has changed
+    - name: Check whether niwot.out has changed
```
Comment (dlebauer) [inline .github/workflows/ci.yml:48]:
```suggestion
    - name: Check whether niwot.out has changed
      shell: bash
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -44,3 +44,14 @@ jobs:
...
+      if: runner.os != 'Windows' #fails b/c line endings?
```
Comment (dlebauer) [inline .github/workflows/ci.yml:49]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -44,3 +44,13 @@ jobs:
       run: |
         echo "::error title={No Output}::Test run for Niwot site failed to produce output"
         exit 1
+    # check whether niwot.out has changed
+    - name: Check whether niwot.out has changed
+      shell: bash
+      run: |
+         if git diff --exit-code Sites/Niwot/niwot.out; then
```
Comment (dlebauer) [inline .github/workflows/ci.yml:51]:
```suggestion
         if git diff --exit-code --ignore-cr-at-eol Sites/Niwot/niwot.out; then
```

**Comment by dlebauer** [comment]
`git diff --exit-code` errored on windows, so excluded it from that job

**Comment by Alomir** [comment]
Maintaining compatibility with Windows will be a never-ending morass of issues; I'm fine with removing the check here.

**Review by Alomir — APPROVED** [review]
Looks good, I'm fine with removing anything beyond strict existence checks from Windows (and even that we might want to drop)

**Comment by dlebauer** [comment]
added removing windows to #19

**Pull request merged**